grafana-plugin/src/containers/UserSettings/parts/connectors/MobileAppConnector.tsx

```diff
@@ -4,30 +4,22 @@ import { Button, Label } from '@grafana/ui';
 import cn from 'classnames/bind';
 
 import { UserSettingsTab } from 'containers/UserSettings/UserSettings.types';
-import { AppFeature } from 'state/features';
-import { useStore } from 'state/useStore';
 
 import styles from './index.module.css';
 
 const cx = cn.bind(styles);
 
-interface SlackConnectorProps {
+interface MobileAppConnectorProps {
   onTabChange: (tab: UserSettingsTab) => void;
 }
 
-const SlackConnector = (props: SlackConnectorProps) => {
+const MobileAppConnector = (props: MobileAppConnectorProps) => {
```

Comment:
rename from `SlackConnector` to `MobileAppConnector` (similarly w/ props interface above)